scripts/tests/test_docker_push_with_retry.py

```
@@ -0,0 +1,250 @@
+  exit 1
+fi
+
+echo "digest: sha256:test size: 1234"
+""",
+      )
+      self._write_executable(fake_bin / "sleep", "#!/usr/bin/env bash\nexit 0\n")
+
+      env = os.environ.copy()
+      env["PATH"] = f"{fake_bin}:{env['PATH']}"
+
+      result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)